### PR TITLE
added and correction Fr ui/eureka/eureka.js

### PR DESCRIPTION
**Pull request by VanSlanzar**
Add missing lines, fix FR NM names based on eureka-tracker

### CODE REVIEW

File: ui/eureka/eureka.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ let Options = {
       'Eureka Pyros': 'Eureka Pyros',
       'Eureka Hydatos': 'Eureka Hydatos',
     },
+    fr: {
+      'Eureka Anemos': 'Eurêka Anemos',
```
[the remark this answers is not in the document]
Comment (VanSlanzar) [reply ui/eureka/eureka.js:31]: I 'll go fix

File: ui/eureka/eureka.js [hunk 2]
```diff
@@ -302,15 +327,15 @@ let Options = {
...
+            fr: 'Jahann',
             ja: 'ジャハ',
             ko: '자하남',
             cn: '台风',
           },
           trackerName: {
             en: 'Jaha',
             de: 'Jaha',
-            fr: 'Jaha',
+            fr: 'Jahann',
```
[the remark this answers is not in the document]
Comment (VanSlanzar) [reply ui/eureka/eureka.js:338]: If I copy / paste, the tracker as you told me, it shows me Jahann. it is a change that has been made. Is that a problem?
NMs non disponibles : Jahann (119m)

**Comment by mooondark** [comment]
Hello there,
The symbol (abbreviation) for minute is "min" not "m" or "mn" (see [https://fr.wikipedia.org/wiki/Minute_(temps)]).

**Comment by VanSlanzar** [comment]
Hi, i put "m" because I'm afraid that with more letters, it goes beyond the frame. I'm going to test in games to see.

**Comment by VanSlanzar** [comment]
Time

Does the frame adjust automatically?

**Comment by mooondark** [comment]
Ouais, tu as raison, il faut un espace.

**Comment by VanSlanzar** [comment]
Je vais regarder ça.
@quisquous, i went spawned Jahannam, the new names work well. I had the impression that there was a doubt. With google translate, I'm not always sure I understand. ^^
1 jahann

2 jahann

**Comment by VanSlanzar** [comment]
Minutes fix

fix min

**Comment by quisquous** [comment]
> @quisquous, i went spawned Jahannam, the new names work well. I had the impression that there was a doubt. With google translate, I'm not always sure I understand. The changes were only for French names ^^.

Sorry for the confusion.  There was no doubt.  I was just saying I that I checked again myself as well and what you wrote looks correct to me.  (Spawning the mob in game is not related to either of those names, so that would not test anything, I don't think?)

**Pull request merged**